static/scss/dashboard.scss

```
@@ -8,6 +8,15 @@ $featureImgMobileWidth: 343px;
     display: flex;
     flex-direction: row;
 
+    &.no-enrollments {
+      flex-direction: column;
+
+      a {
+        color: black;
+        text-decoration: underline;
+      }
+    }
```

The `<p>` tag is adding extra space to the bottom of this card. Can you add a rule to remove that margin under the `&.no-enrollments` selector?

```suggestion
    &.no-enrollments {
      flex-direction: column;

      a {
        color: black;
        text-decoration: underline;
      }
      
      p {
        margin: 0;
      }
    }
```